### ISSUE TITLE
Uninstall 

### ISSUE THREAD

**Issue by Jyosuke-Kun** [st2024/Steamtools#38]
I want to uninstall it so how can i

**Comment by AkinoStyx** [comment]
yeah i was also wondering the same thing how can i uninstall steamtools if i dont want it  in future
